.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,15 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
What is this .idea folder for and these files for? I have encountered them for the first time while doing a Django project.